framework/src/base/Adaptivity.C

```diff
@@ -133,7 +135,7 @@ bool
 Adaptivity::adaptMesh()
 {
   bool meshChanged = false;
-  if (_mesh_refinement_on && (_start_time <= _t && _t < _stop_time))
+  if (_mesh_refinement_on && (_start_time <= _t && _t < _stop_time) && _step % _interval == 0)
```

Comment:
No I'm not, It looks like that would be the case. We don't flush there so
is it getting output with other stuff and you don't see it?

On Wed, May 4, 2016 at 8:46 AM Daniel Schwen <notifications@github.com>
wrote:

> In framework/src/base/Adaptivity.C
> <https://github.com/idaholab/moose/pull/6892#discussion_r62051868>:
>
> > @@ -133,7 +135,7 @@ bool
> >  Adaptivity::adaptMesh()
> >  {
> >    bool meshChanged = false;
> > -  if (_mesh_refinement_on && (_start_time <= _t && _t < _stop_time))
> > +  if (_mesh_refinement_on && (_start_time <= _t && _t < _stop_time) && _step % _interval == 0)
>
> Are you sure? That is not what I observed when I ran my tests with the
> interval enabled.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6892/files/a81e03a99421460105829d94f8a5e88eab8487f4#r62051868>
>